Event/AdminEvent.php

```diff
@@ -13,6 +13,13 @@ class AdminEvent extends Event
     protected $configuration;
 
     /**
+     * Related Admin name.
+     *
+     * @var string
+     */
+    protected $adminName;
```

Comment:
Yes, it is used in Admin:generateRouteName method, and Action class too. It is more convenient to store it, instead of getting configuration each time.